daml-lf/encoder/src/test/scala/com/digitalasset/daml/lf/archive/testing/DamlLfEncoderTest.scala

```diff
@@ -71,7 +73,18 @@ class DamlLfEncoderTest
     for {
       pkgWithId <- dar.main +: dar.dependencies
       (_, pkg) = pkgWithId
```

Comment:
Maybe just inline this destructuring.

Reply:
No, don't do that. Even exhaustive patterns on the LHS of `<-` entail a `withFilter` call. We separate destructuring pretty universally for this reason.